addons/mail/static/src/core/common/thread_actions.js

```diff
@@ -7,17 +7,44 @@ import { registry } from "@web/core/registry";
 
 export const threadActionsRegistry = registry.category("mail.thread/actions");
 
-threadActionsRegistry.add("close", {
-    condition(component) {
-        return component.props.chatWindow;
-    },
-    icon: "fa fa-fw fa-close",
-    name: _t("Close Chat Window"),
-    open(component) {
-        component.close();
-    },
-    sequence: 100,
-});
+threadActionsRegistry
+    .add("toggle", {
```

Comment:
I prefer "fold-chat-window" as the action name. "Toggle" is too broad, especially when a subset of actions are toggleable (the ones that show a dedicated panel, e.g. `pinned-messages`